src/lib/Microsoft.Fx.Portability/ObjectModel/AnalyzeResponse.cs

```diff
@@ -1,50 +1,14 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.Fx.Portability.Reporting.ObjectModel;
-using Newtonsoft.Json;
 using System;
-using System.Collections.Generic;
-using System.Runtime.Versioning;
 
 namespace Microsoft.Fx.Portability.ObjectModel
 {
-    public sealed class AnalyzeResponse : IComparable
+    public sealed class AnalyzeResponse
```

Comment:
Hmm, why did we remove IComparable?

Reply:
The new class is never used as `IComparable`.